dsl/camel-jbang/camel-jbang-plugin-kubernetes/pom.xml

```diff
@@ -54,6 +54,13 @@
             <version>${kubernetes-client-version}</version>
         </dependency>
 
+        <!-- Kubernetes -->
+        <dependency>
```

Comment:
Why is this JAR added ?

Reply:
I mean did this JAR bring any code that this plugin needs or was it added for some other reason? Those JARs are a bit big so we should avoid if not needed.

Reply:
It brings the Route classes for the Openshift Route ressource, but I see what you means. Let me check if the model jar is enough in our case.

Reply:
I replaced the client jar with the model jar.